threshold_tools_application_examples.ipynb

```diff
@@ -1143,7 +1090,54 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "app.export_dataset(return_period,'my_filename')"
+    "app.export_dataset(return_period, 'my_filename')"
```

Comment:
There's no object `return_period` in this notebook - should just change this to one of the data objects in this notebook

Reply:
Well spotted! Yes, maybe we just delete these two cells?

Reply:
Done! Also modified the markdown for this - but need @bethmints confirmation on this too